### PR TITLE
Adding compatibility with brux cluster

### PR DESCRIPTION
**Pull request by mondalspandan**

### CODE REVIEW

File: runner.py [hunk 1]
```diff
@@ -670,10 +688,15 @@ def get_main_parser():
                 job_script_prologue=job_script_prologue,
             )
         elif "condor" in args.executor:
+            portopts = {}
+            if "brux" in args.executor:
+                import socket
+                portopts = {"port": 8786, "host": socket.gethostname()}
```
Comment (Ming-Yan) [inline runner.py:694]: Indeed I think we can move the config for lxplus out (after [L638](https://github.com/cms-btv-pog/BTVNanoCommissioning/pull/58/files#diff-34cde5e49b54d9b67952c80d3580b6c421ebcf16097996e215aef54e44f78562R638)

https://github.com/cms-btv-pog/BTVNanoCommissioning/pull/58/files#diff-34cde5e49b54d9b67952c80d3580b6c421ebcf16097996e215aef54e44f78562R650-R655

which is also used here

File: runner.py [hunk 2]
```diff
@@ -670,10 +688,15 @@ def get_main_parser():
...
+            portopts = {}
+            if "brux" in args.executor:
+                import socket
+                portopts = {"port": 8786, "host": socket.gethostname()} 
             cluster = HTCondorCluster(
                 cores=args.workers,
                 memory=f"{args.memory}GB",
                 disk=f"{args.disk}GB",
+                scheduler_options=portopts,
                 job_script_prologue=job_script_prologue,
```
Comment (Ming-Yan) [inline runner.py:700]: Please also add the ` job_extra_directives`  https://github.com/cms-btv-pog/BTVNanoCommissioning/pull/58/files#diff-34cde5e49b54d9b67952c80d3580b6c421ebcf16097996e215aef54e44f78562R667-R674

**Review by Ming-Yan — CHANGES_REQUESTED** [review]
Thanks @mondalspandan for the PR and adding new cluster in the test!

the PR looks good, I only have some minor comments inline.

Also can you please do `black .` to make the linting works ?

File: src/BTVNanoCommissioning/utils/correction.py
[the remark this answers is not in the document]
Comment (mondalspandan) [reply src/BTVNanoCommissioning/utils/correction.py:1]: I was experimenting with some variable paths, forgot to remove the f

File: runner.py (hunk 2 above)
Comment (mondalspandan) [reply runner.py:707]: It ran on brux without `job_extra_directives`. Probably not needed

**Review by Ming-Yan — APPROVED** [review]
Thanks for the update! I would merge this PR to the main repo!

**Pull request merged**